db/1.14/modules/ROOT/pages/database-connector-input-output-parameters.adoc

```diff
@@ -0,0 +1,219 @@
+----
+
+== Configure In Out Parameters
+
+Consider a scenario where you develop a Mule app for an inventory management system to handle the process of updating the stock levels for specific products. The Mule app interacts with a stored procedure named `UpdateStockLevel` in a PostgreSQL database, which requires both input and output parameters. This following example illustrates how to configure the *In out parameters* field of the *Stored procedure* operation to manage the stock level updates, including both the input data (product ID and quantity) and the output data (updated stock level).
+
+. In Studio, add a Database Configuration element to your MuleSoft app project and configure it with the necessary PostgreSQL database connection details.
+. Drag the *Stored procedure* operation to your canvas that retrieves the order information.
+. Select the operation from your flow.
+. In the *SQL Query text* field, enter `{call UpdateStockLevel(:productId, :quantity, :updatedStockLevel)}`.
++
+This defines the SQL statement for calling a stored procedure UpdateStockLevel with input parameters productId and quantity, and an output parameter updatedStockLevel.
+. Set *In out parameters* to *Edit inline*.
+. Click the plus icon to open the *In out parameter* window.
+. In the *In out parameter* window, set the following parameters:
++
+[%header%autowidth.spread]
+|===
+|Key | Value
+|productId | #[payload.productId]
+|quantity | #[payload.quantity]
+|updatedStockLevel | (empty)
+|===
++
+[start=8]
+. Click *Finish*.
+
+The configuration enables you to execute the stored procedure, providing input parameters product ID and quantity, and retrieving the updated stock level as an output parameter.
+
+In the *Configuration XML* editor, the configuration looks like this:
+[source,xml,linenums]
+----
+<db:stored-procedure doc:name="Stored procedure"			
+<db:sql ><db:in-out-parameters >
+			</db:in-out-parameters><![CDATA[{call UpdateStockLevel(:productId, :quantity, :updatedStockLevel)}]]>
+			</db:sql>
+			<db:in-out-parameters >
+				<db:in-out-parameter key="updatedStockLevel" />
+				<db:in-out-parameter key="productId" value="#[payload.productId]" />
+				<db:in-out-parameter key="quantity" value="#[payload.quantity]" />
+			</db:in-out-parameters>
+----
+
+== Configure Custom Types
+
+The ability to use custom types as parameters in stored procedures depends on the database system you are using and the capabilities of the Database Connector or framework you are working with. 
+Exercise caution when you work with an Oracle database vendor and use the *Stored procedure* operation with custom database types. 
+When you define the stored procedure within a package, set the column type of a custom type parameter in the connector connection settings:
+
+. In Studio, open the Database Config Global Element Properties window.
+. Click the *Advanced* tab.
+. Set *Column types* to *Edit inline*.
+. Click the plus sign to open the *Column types* window.
```

Comment:
Sometimes you bold window names and sometimes you don't. This should be consistent.